public/client.js

```diff
@@ -155,97 +145,22 @@ socket.on('user join', (data) => {
     }
 });
 
-//displays message to users
+// displays message to users
+// TODO: this method is GIANT and should be broken into smaller parts
+// It's also very brittle, relying on the format of the msg
```

Comment:
Now you have achieved this task, so you can remove these comments.

Reply:
👍